### ISSUE TITLE
Questions about training

### ISSUE THREAD

**Issue by lensea** [shekkizh/FCN.tensorflow#108]
Implementation environment : python3.6 tensorflow 1.8

1. When I run the program exactly according to your code, the single image in the validation set will not reach your effect in the logs/images directory.
2. I used tf.layers to rebuild the network and added the bn relu layer in the first of the network.
My Lr=1e-5, RMSP modify lr  and not use pre-train 
About after 5 epoch ,the loss from  5.0+ down to 1.0. It is basically the same as the loss map that you gave.
But when I tested it, I found that the result of the segmentation was very poor. It doesn't work for you at all.
Can you help me analyze the reasons? tks

my result as follow:
![image](https://user-images.githubusercontent.com/9279010/50435852-0d667c80-091e-11e9-83b2-2225e6d7b5e4.png)
